source/atlas-search.txt

```diff
@@ -18,4 +18,86 @@ Atlas Search
    :depth: 2
    :class: singlecol
 
-See :atlas:`Atlas Search </atlas-search/>` in the MongoDB Atlas documentation.
\ No newline at end of file
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to
+run :atlas:`Atlas Search </atlas-search/>` queries on a collection.
+Atlas Search enables you to perform full-text searches on collections
+hosted on MongoDB Atlas. Atlas Search indexes specify the behavior of the
+search and which fields to index.
+
+Sample Data
+~~~~~~~~~~~
+
+The example in this guide uses the ``movies`` collection in the ``sample_mflix``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to
+create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Run an Atlas Search Query
+-------------------------
+
+This section shows how to create an aggregation pipeline to run an
+Atlas Search on a collection. You can use the ``Aggregates.search()`` builder
+method to create a ``$search`` pipeline stage, which specifies the search
+criteria. Then, call the ``aggregate()`` method and pass your pipeline as a parameter.
+
+.. tip:: 
+
+   To learn more about aggregation operations and builders, see the :ref:`java-aggregation`
+   guide. 
+
+Before running an Atlas Search query, you must create an Atlas Search index
+on your collection. To learn how to programmatically create an Atlas Search
+index, see the :ref:`java-search-indexes` section in the Indexes guide. 
+
+Atlas Search Example
+~~~~~~~~~~~~~~~~~~~~
+
+This example performs an Atlas Search query by calling the
+``aggregate()`` method and passing the following pipeline stage
+builders:
```

Comment:
_"passing the following pipeline stage builders"_

1. We are talking about builder methods here, not builders. The latter usually used to refer to a builder object, which is not involved here.
2. The example does not pass a builder / builder method anywhere, it passes the results of calling/invoking those methods.